src/napari_blob_detection/_widget.py

```diff
@@ -39,6 +39,11 @@ def detect_blobs(
     all_sigmas = []
     algorithm_func = ALGORITHM_LOOKUP[algorithm]
     ndim = DIMENSIONALITY_LOOKUP[dimensionality]
+    if ndim == 2:
+        radius = np.sqrt(2)
+    else:
+        radius = np.sqrt(3)
```

Comment:
```suggestion
    radius = np.sqrt(ndim)
```

Given how dimensionality stuff is defined right now, this code should be equivalent. I think we'll probably only support 2/3 dimensional data (in terms of what is passed into scikit-image), so I think it should be fine to define it like that in the future too.